dolfyn/tools/misc.py

```diff
@@ -356,18 +312,8 @@ def medfiltnan(a, kernel, thresh=0):
     out = medfilt2d(a, kernel)
     if thresh > 0:
         out[convolve2d(np.isnan(a),
-                       np.ones(kernel) / np.prod(kernel),
-                       'same') > thresh] = np.NaN
+                        np.ones(kernel) / np.prod(kernel),
+                        'same') > thresh] = np.NaN
     if flag_1D:
         return out[0]
     return out
-
-
-def degN2cartDeg(angN,):
```

Comment:
This is a useful little helper function. Let's keep it around.

Reply:
Done